.stylelintrc

```diff
@@ -6,6 +6,7 @@
   "rules": {
     "custom-property-pattern": "^(?!.+)",
     "scss/comment-no-empty": null,
+    "scss/operator-no-newline-after": null,
```

Comment:
Disabling this rule, because it conflicts with prettier formatting.

Ideally we should upgrade to stylelint v15, where they aligned with prettier by default: https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules